File: content/code/count_word_occurrence_in_string_sol.jl
@testset "Test count word occurrence in string" begin
    @test count_word_occurrence_in_string("AAA BBB", "AAA") == 1
    @test count_word_occurrence_in_string("AAA AAA", "AAA") == 2
    @test count_word_occurrence_in_string("AAAAA", "AAA") == 1
end

File: content/code/count_word_occurrence_in_string_sol.R
test_that("Test count word occurrence in string", {
  expect_equal(count_word_occurrence_in_string("AAA BBB", "AAA"), 1)
  expect_equal(count_word_occurrence_in_string("AAA AAA", "AAA"), 2)
  expect_equal(count_word_occurrence_in_string("AAAAA", "AAA"), 0)
})
